ecosystem/sep-0038.md

```diff
@@ -152,15 +152,15 @@ Name | Type | Description
       "delivery_methods": [
         {
           "name": "cash",
-          "description": "Pick up cash BRL at one of our payout locations."
+          "description": "Deposit cash BRL to one of our payout locations."
```

Comment:
Payout = withdraw, so is this actually a deposit or is this a withdrawal? Payout might be the wrong term.

Reply:
How does this sound? https://github.com/stellar/stellar-protocol/pull/987/commits/023316ccd06f76af962074c941e0c7db148c1d17